common/src/main/java/me/shedaniel/architectury/event/events/ChatEvent.java

```diff
@@ -27,11 +27,21 @@
 
 public interface ChatEvent {
     /**
-     * Invoked when server receives a message, equivalent to forge's {@code ServerChatEvent}.
```

Comment:
Here as well

Reply:
The sentence was moved to the interface javadoc